boards/adafruit/feather_nrf52840/adafruit_feather_nrf52840_flash.dtsi

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 Jacob Winther
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+};
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x0000C000>;
+		};
+		slot0_partition: partition@c000 {
+			label = "image-0";
+			reg = <0x0000C000 0x00076000>;
+		};
+		slot1_partition: partition@82000 {
+			label = "image-1";
+			reg = <0x00082000 0x00076000>;
```

Comment:
```suggestion
			reg = <0x0000C000 0x00077000>;
		};
		slot1_partition: partition@82000 {
			label = "image-1";
			reg = <0x00082000 0x00075000>;
```

Reply:
I'd copied that from the 52840dk. Does you change mean that the two partitions will be different sizes?

Reply:
If you use swap using move, you need a sector for the data to be moved up by, and you need space for the swap status fields, which is about a sector, so by making the changes here you get the full 0x65000 for an image, without these changes you get 0x64000

Reply:
Got it, thanks for explaining.

Reply:
I adjusted the start for slot1 to match.

Should the 52840dk be updated with the same changes? Partly asking as I have another PR (#77835) to move this to common.

Reply:
Think you've updated the wrong board